src/main/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilder.java

```diff
@@ -161,11 +164,38 @@ public Query generateRollupCountsQueryForCriteriaGroupsList(
             null,
             true);
 
-    FilterVariable filterVariable =
-        new BooleanAndOrFilterVariable(
-            BooleanAndOrFilterVariable.LogicalOperator.OR,
-            criteriaGroupsList.stream().map(this::generateFilterForCriteriaGroups).toList());
+    FilterVariable filterVariable = generateFilterVariable(criteriaGroupsList);
```

Comment:
Now that this is so short, how would you feel about inlining `generateFilterVariable(criteriaGroupsList)`?